docs/conf.py

```diff
@@ -6,24 +6,23 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'scores'
-copyright = '2023, Australian Bureau of Meteorology'
-author = 'Australian Bureau of Meteorology'
-release = '0.4'
+project = "scores"
+copyright = "2023, Australian Bureau of Meteorology"
+author = "Australian Bureau of Meteorology"
+release = "0.4"
```

Comment:
Hey @John-Sharples could you  also add the scores version to the sphinx docs. I'm not doing a review, just thought its something you could add since your currently working on it.
```suggestion
release = "0.4"

from scores import __version__
version = __version__
```

Reply:
Can probably replace the release variable for the version.